mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,16 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.eraseOrder = async (req, res) => {
+  Order.findByIdAndDelete(req.params.orderId)
+    .then(deletedOrder => {
+      if (!deletedOrder) {
+        return res.status(404).json({ error: "No such order found" });
+      }
+      res.json({ success: "Order has been deleted" });
```

Comment:
The response object should be consistent across the API. In this case, the error responses are in the format `{ error: errorHandler(err) }`, but the success response is in the format `{ success: "Order has been deleted" }`. It should be consistent, either `{ success: "Order has been deleted" }` or `{ message: "Order has been deleted" }`.